pkg/apis/scheduling/types.go

```diff
@@ -111,6 +114,12 @@ const (
 
 	// NotEnoughPodsReason is probed if there're not enough tasks compared to `spec.minMember`
 	NotEnoughPodsReason string = "NotEnoughTasks"
+
+	// NotEnoughPodsOfTaskReason is probed if there're not enough pods of task compared to `spec.minTaskMember`
+	NotEnoughPodsOfTaskReason string = "NotEnoughPodsOfTask"
+
+	// NotInqueueableReason if probed if job is rejected to enqueue
+	NotInqueueableReason string = "NotInqueueable"
```

Comment:
`NotInqueueableReason string = "NotInqueueable"` to UnEnqueueableReason string = "Unenqueueable"